.github/ISSUE_TEMPLATE/feature_request_template.yaml

```
@@ -0,0 +1,118 @@
+name: Feature Request
+description: Use this template for feature and enhancement proposals.
+labels: 'enhancement'
+title: '[FR]'
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for submitting a feature request. **Before proceeding, please review MLflow's [Issue Policy for feature requests](https://www.github.com/mlflow/mlflow/blob/master/ISSUE_POLICY.md#feature-requests) and the [MLflow Contributing Guide](https://github.com/mlflow/mlflow/blob/master/CONTRIBUTING.rst)**.
+        **Please fill in this feature request template to ensure a timely and thorough response.**
+  - type: dropdown
+    id: contribution
+    attributes:
+      label: Willingness to contribute
+      description: The MLflow Community encourages new feature contributions. Would you or another member of your organization be willing to contribute an implementation of this feature (either as an MLflow Plugin or an enhancement to the MLflow code base)?
+      options:
+        - Yes. I can contribute this feature independently.
+        - Yes. I would be willing to contribute this feature with guidance from the MLflow community.
+        - No. I cannot contribute this feature at this time.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Proposal Summary
+      description: |
+        In a few sentences, provide a clear, high-level description of the feature request
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Motivation
+      description: |
+        - What is the use case for this feature?
+        - Why is this use case valuable to support for MLflow users in general?
+        - Why is this use case valuable to support for your project(s) or organization?
+        - Why is it currently difficult to achieve this use case? (please be as specific as possible about why related MLflow features and components are insufficient)
+      value: |
+        ### What is the use case for this feature?
+
+        ### Why is this use case valuable to support for MLflow users in general?
+
+        ### Why is this use case valuable to support for your project(s) or organization?
+
+        ### Why is it currently difficult to achieve this use case?
```

```suggestion
        #### What is the use case for this feature?

        #### Why is this use case valuable to support for MLflow users in general?

        #### Why is this use case valuable to support for your project(s) or organization?

        #### Why is it currently difficult to achieve this use case?
```

because the section header (`Motivation`) uses `###`.